Text document as basis for changes

capzDE opened the pull request:
The most crucial aliases already combined to give a basis for further changes and provide an idea for structuring of the alias files.

capzDE committed:
Text document as basis for changes

The most crucial aliases already combined to give a basis for further changes and provide an idea for structuring of the alias files.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.pn Pushback approved, face North.
+.pe Pushback approved, face East.
+.ps Pushback approved, face South.
+.pw Pushback approved, face West.
```
Würde definitiv noch .sp .spe etc. für gleichzeitiges startup und pushback hinzufügen.

capzDE replied:
> Würde definitiv noch .sp .spe etc. für gleichzeitiges startup und pushback hinzufügen.

Hatte mir gedacht, dass man bei einer kombinierten Anweisung ja auch einfach .su Leertaste .p Leertaste machen könnte und dann ja auch beide hat - wäre dann auch ein wenig aufgeräumter (und weniger ist ja bekanntlich mehr) - aber spricht an sich natürlich nichts dagegen.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.ps Pushback approved, face South.
+.pw Pushback approved, face West.
+.hp Hold position.
+.t Taxi to holding point runway $deprwy via $uc($1).
```
Finde .t etwas unintuitiv. Aus meiner sicht wäre da ein generelles Taxi to $(uc$1) via $(uc$2) besser, Beispiel München taxi to entry. Dagegen .tr für taxi to holding point.

a3li replied:
Die Entries brauchen wir so oft, da wird unser `.te` bestehen bleiben, ein `.t Entry` ist da zu umständlich.

capzDE replied:
> Die Entries brauchen wir so oft, da wird unser `.te` bestehen bleiben, ein `.t Entry` ist da zu umständlich.

Dann würde ich sagen, machen wir am besten .te generell rein. Ich denke die Berliner werden das auch haben wollen.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.t Taxi to holding point runway $deprwy via $uc($1).
+.ts Taxi to stand $uc($1) via $uc($2).
+.th Taxi via $uc($1), hold short of $uc($2).
+.tc Continue taxi.
```
Finde es nicht gut, in der Abkürzung die Wortreihenfolge zu vertauschen, aus meiner Sicht zu unintuitiv. Fände .ct besser und v.A. besser einprägbar

capzDE replied:
> Finde es nicht gut, in der Abkürzung die Wortreihenfolge zu vertauschen, aus meiner Sicht zu unintuitiv. Fände .ct besser und v.A. besser einprägbar

Meine Idee war ein Prefix-Suffix System, sodass die Art der Anweisung/Sprechgruppe am Anfang steht und danach wo nötig ein entsprechender Suffix für eventuelle Modifikationen. In diesem Fall also .t für alle Taxianweisungen und das c dahinter spezifiziert, dass es um die continue Anweisung gehen soll.
Persönlich fände ich das so langfristig intuitiver und eindeutiger (auch wenn Leute später in der Aliasdatei suchen, weil sie dann einfach nur schauen müssen, was - um beim Beispiel zu bleiben - mit .t anfängt; besonders wenn man das alphabetisch sortiert, wäre das glaube ich unglaublich sinnvoll).

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.ctg Wind $winds, runway $arrrwy, cleared touch and go.
+.cla Wind $winds, runway $arrrwy, cleared low approach.
+.rdy Report ready for departure.
+.rdyimm Are you ready for immediate departure?
```
Bisschen langer alias unter Stress imo

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.cla Wind $winds, runway $arrrwy, cleared low approach.
+.rdy Report ready for departure.
+.rdyimm Are you ready for immediate departure?
+.stop Stop immediately.
```
Fände mit wiederholung besser.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.rdy Report ready for departure.
+.rdyimm Are you ready for immediate departure?
+.stop Stop immediately.
+.ga Go around, follow published missed approach procedure.
```
follow published missed approach unnötig. dann könnte man sich auch das .gavec und .garwy sparen, zu spezifische alias. Man kan ja einfach .ga und dann einen heading vector alias verwenden. An so einen alias würde sich niemand erinnern.

capzDE replied:
> follow published missed approach unnötig. dann könnte man sich auch das .gavec und .garwy sparen, zu spezifische alias. Man kan ja einfach .ga und dann einen heading vector alias verwenden. An so einen alias würde sich niemand erinnern.

Finde ich eine gute Idee, würde es aufgeräumter machen.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.sqc Squawk Charlie.
+.sqs Squawk Standby.
+.id Identified.
+.c Climb flight level $uc($1).
```
würde .cfl oder .cl besser finden (obwohl natürlich verwechslungsgefahr mit cleared flight level)

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.daq Descend altitude $uc($1)ft, QNH $altim($arr).
+.dva Descend via STAR flight level $uc($1).
+.dtr Descend to reach flight level $uc($1) at $uc($2).
+.roc Rate of climb $uc($1)ft per minute.
```
falsche phraseo, lieber climb at ... feet per minute, außerdem wäre or greater or less praktisch

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.hold Hold over $uc($1) as published.
+.holdifl Hold over $uc($1), inbound track $uc($1) degrees, $2 hand turns, $uc($3) minute legs.
+.h Fly heading $uc($1).
+.hl Turn left heading $uc($1).
```
Fände .tl als Turn Left intuitiver

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.spdm Mach $uc($1).
+.spdfa Reduce final approach speed.
+.spdn Resume normal speed.
+.spdh High speed approved.
```
imo lieber .nos und .hsa

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.cvis Cleared visual approach runway $arrrwy.
+.spd Maintain speed $uc($1) knots.
+.spdu Maintain speed $uc($1) knots until $uc($2) miles final.
+.spdm Mach $uc($1).
```
.spdm finde ich komisch, dann lieber nur .m

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.ra Report present altitude.
+.rf Report present flight level.
+.rh Report present heading.
+.ei Expect ILS approach runway $arrrwy.
```
finde das aktuell .exp für alles expect sehr praktisch. Da könnte man dann auch andere Expect Dinge dranhängen.

capzDE replied:
> finde das aktuell .exp für alles expect sehr praktisch. Da könnte man dann auch andere Expect Dinge dranhängen.

Ist halt die Frage was im Zweifel einfacher ist. .exp und dann muss man den Approach Type selber noch manuell eintragen, oder ein eigener Alias pro Fall.

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.rod Rate of descent $uc($1)ft per minute.
+.qnh QNH $altim($arr).
+.hold Hold over $uc($1) as published.
+.holdifl Hold over $uc($1), inbound track $uc($1) degrees, $2 hand turns, $uc($3) minute legs.
```
imo unmerkbar

AntonBeasis commented on alias revised.txt:
```diff
@@ -0,0 +1,171 @@
+.pd Proceed direct $uc($1).
+.star Cleared $star arrival.
+.trans Cleared $star transition.
+.ri Report indicated airspeed.
```
.reps fände ich intuitiver (siehe .exp). Könnte man dann auch mit z.B. repint report intentions oder so wiederverwenden und wäre imo einfacher zu merken.

AntonBeasis reviewed (commented):
Ganz gute Basis, auch wenn an einigen Stellen aus meiner Sicht etwas verkompliziert und daher schwierig zu merken.

capzDE committed:
Update alias revised.txt

Added .te (taxi to entry) alias

Phil061995 committed:
Update alias revised.txt

Phil061995 committed:
Update alias revised.txt

Phil061995 committed:
Update alias revised.txt

capzDE committed:
Link for vACDM pilot guide added

capzDE committed:
Changed vACDM aliases

.vacdm would interfere with the plugin commands, which also use .vacdm, so the contents have been moved to .tobt

Frenkii reviewed (approved):
inital baseline.
